addons/project/views/project_task_views.xml

```diff
@@ -1233,4 +1285,20 @@
             </field>
         </record>
 
+        <record id="project_milestone_action_view_tasks" model="ir.actions.act_window">
+            <field name="name">Tasks.test</field>
+            <field name="res_model">project.task</field>
+            <field name="view_mode">kanban,list,form,calendar,pivot,graph,activity</field>
+            <field name="context">{'default_project_id': active_id}</field>
+            <field name="view_ids" eval="[(5, 0, 0),
+                (0, 0, {'view_mode': 'kanban', 'view_id': ref('project.project_task_kanban_view_project_milestone')}),
+                (0, 0, {'view_mode': 'list', 'view_id': ref('project.project_task_tree_view_project_milestone')}),
+                (0, 0, {'view_mode': 'form'}),
+                (0, 0, {'view_mode': 'calendar'}),
+                (0, 0, {'view_mode': 'pivot', 'view_id': ref('project.project_task_pivot_view_project_milestone')}),
+                (0, 0, {'view_mode': 'graph', 'view_id': ref('project.project_task_graph_view_project_milestone')}),
+                (0, 0, {'view_mode': 'activity'})
+            ]"/>
+        </record>
```

Comment:
We can create `ir.actions.act_window.view` to set the view we want in the action instead of doing as you did. 

With your current changes, we cannot easily change a view or add a new view without resetting the `view_ids` field. 😕